bridges/snowbridge/primitives/router/src/outbound/mod.rs

```diff
@@ -407,3 +413,42 @@ where
 		Ok((Command::MintForeignToken { token_id, recipient, amount }, *topic_id))
 	}
 }
+
+/// An adapter for the implementation of `ExporterFor`, which attempts to find the
+/// `(bridge_location, payment)` for the requested `network` and `remote_location` and `xcm`
+/// in the provided `T` table containing various exporters.
+pub struct NetworkWithXcmExportTable<T, M>(core::marker::PhantomData<(T, M)>);
+impl<T: Get<Vec<NetworkExportTableItem>>, M: Contains<Xcm<()>>> ExporterFor
```

Comment:
Make T implement `ExporterFor` directly removing dependency on `NetworkExportTableItem`.
```suggestion
impl<T: ExporterFor, M: Contains<Xcm<()>>> ExporterFor
```

Reply:
https://github.com/paritytech/polkadot-sdk/pull/6074/commits/a6dbcbacd0bde44b8a6cf85667657d7c578a2c2d